week_3/Binary_Tree/invert_binary_tree/invert_binary_tree.js

```diff
@@ -0,0 +1,28 @@
+// Given the root of a binary tree, invert the tree, and return its root.
+
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.left = (left===undefined ? null : left)
+ *     this.right = (right===undefined ? null : right)
+ * }
+ */
+/**
+ * @param {TreeNode} root
+ * @return {TreeNode}
+ */
+const invertTree = (root) => {
+	if (root == null) {
+		return root;
+	}
+	invertTree(root.left);
+	invertTree(root.right);
+
+	let temp = root.left;
```

Comment:
```suggestion
	const temp = root.left;
```

always default to `const` and let the compiler yell at your if you try to mutate it 😄